scripts/readelf.py

```diff
@@ -1064,12 +1102,18 @@ def _dump_debug_frames_interp(self):
                         fullhex=True, lead0x=False)))
                 ra_regnum = entry.cie['return_address_register']
 
+            else: # ZERO terminator
+                assert isinstance(entry, ZERO)
```

Comment:
I'm not sure about the name ZERO here; maybe END_MARKER? Is this dictated by the output of readelf?

Reply:
ZERO is indeed what `readelf` displays:

```sh
$ readelf --debug-dump=frames /bin/ls|tail -n7
  DW_CFA_nop
  DW_CFA_nop
  DW_CFA_nop

00002cf0 ZERO terminator
```

As it’s not present in DWARF and as there is no documentation for this, I would suggest we stick to `readelf`’s naming, to reduce the possible confusion. What do you think?

Reply:
Sounds good, but please beef up the comment in the place where it's defined to explain the name choice